{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -73,6 +73,7 @@
 INSTALLED_APPS += ["storages"]  # noqa F405
 {%- endif -%}
 {% if cookiecutter.cloud_provider == 'AWS' %}
+INSTALLED_APPS += ["health_check.contrib.s3boto_storage"]
```

Comment:
Should not it be `"health_check.contrib.s3boto3_storage"`?

Reply:
Feel free to propose the change in this PR and I will update it

Reply:
Sorry, this PR has conflict and cannot be patched from my fork. A "PRable" branch is https://github.com/BoPeng/cookiecutter-django/tree/health , which uses `health_check.contrib.s3boto3_storage` because this is what works for me.